internal/configurations/assets.ps1

```diff
@@ -0,0 +1,6 @@
+<#
+This is designed for all online assets you need configurations for.
+#>
+
+# The default path where dbatools stores persistent data
+Set-DbaConfig -Name 'assets.sqlbuildreference' -Value 'https://sqlcollaborative.github.io/assets/dbatools-buildref-index.json' -Default -DisableHandler -Description "The url where dbatools fetches the up to date builreference index"
```

Comment:
Typo: `Builreference`
You may also want to include the product the build reference is for in the description (just to be 100% blunt and direct)